lib/helpers/users.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+Future addUsers(allUsers) async {
```

Comment:
Any helper operations related to users would be placed here (e.g. `deleteUser` or `getUser`)